lib/addons/p5.dom.js

```diff
@@ -242,6 +242,139 @@
     }
   };
 
+  // General handler for event attaching and detaching
+  function adjustListener(ev, fxn, ctx) {
```

Comment:
are these three methods duplicated from p5.Element.js? if so, let's attached these functions to p5.Element as p5.Element_adjustListener and p5.Element._attachListener rather than duplicate the code in two places.